src/igor/config.go

```diff
@@ -77,6 +77,9 @@ type Config struct {
 
 	// Domain for email address
 	Domain string
+
+	//Igor Notify Notice time (in hours) is the amount of time before reservation expires users are notified
+	ExpirationLeadTime int
```

Comment:
Because all of our other duration values are in minutes, can you please convert this value to minutes for the sake of consistency?

Reply:
It looks like the notes still indicate that the value should be in hours. Can you please update?